F-FrontEnd/gnu_module/trans_module.c

```diff
@@ -766,7 +766,7 @@ static ID  make_id( gfc_symbol *sym , const char *name )
 
        if (sym->attr.flavor == FL_MODULE) {
    
-          id->class = CL_MODULE;
+          id->cls = CL_MODULE;

```

Comment:
Is this change for avoiding a C++ keyword "class"?

Reply:
yes